### ISSUE TITLE
New Feature: Subscribe to specific TAGs

### ISSUE THREAD

**Issue by jbasila** [caelum/mamute#262]
I would like to see a feature of allowing users to subscribe to specific TAGs. When a question is posted with a specific TAG, the users that are subscribed to the specific TAG will get a notification.
This will allow users to not get flooded with questions that are irrelevant to them and only focus on what they are able to answer.

It's also possible to add the TAGs to the notification e-mail in which the users can then use their filter mail rules to filter out questions based on TAGs.

**Comment by felipeweb** [comment]
@jbasila  Do you want to send a PR with this new feature?

**Comment by BMAnush** [comment]
Yup, this feature will be really helpful. Can we have this feature in future releases??
